config/environments/production.rb

```diff
@@ -49,9 +49,6 @@
   # Generate digests for assets URLs
   config.assets.digest = true
 
-  # add special CSS files to compile
-  config.assets.precompile += %w(ckeditor/* application_* .svg)
```

Comment:
Oops only the ckeditor part of this should be removed. `application_*` and `.svg` should stay.